### PR TITLE
HCIDOCS-340: Added a release note regarding the new HostFirmwareComponents resource

### PR DESCRIPTION
**Pull request by johnwilkins**
Adds a release note about the HostFirmwareComponents resource.

Version(s): 4.16

Issue: https://issues.redhat.com/browse/HCIDOCS-340

Link to docs preview: https://77386--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-upgrading-or-downgrading-host-firmware

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by jeana-redhat — COMMENTED** [review]
A nit and a half :)

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -240,6 +240,11 @@ These additions help identify issues that might stem from using a specific versi
 
 For more information, see xref:../support/gathering-cluster-data.adoc#about-must-gather_gathering-cluster-data[About the must-gather tool].
 
+[id="ocp-4-16-upgrading-or-downgrading-host-firmware"]
+==== Upgrading or downgrading host firmware
+
+Beginning with {product-title} {product-version}, you can upgrade or downgrade a bare-metal node's firmware to a specific version. Metal^3^ provides the `HostFirmwareComponents` resource, which describes BIOS and baseboard management controller (BMC) firmware versions. Upgrading or downgrading firmware is useful when deploying an {product-title} cluster on bare metal with validated patterns that have been tested against specific firmware versions. See xref:../post_installation_configuration/bare-metal-configuration.adoc#about-the-hostfirmwarecomponents-resource_post-install-bare-metal-configuration[About the HostFirmwareComponents resource] for additional details.
```
[the remark this answers is not in the document]
Comment (johnwilkins) [reply release_notes/ocp-4-16-release-notes.adoc:246]: That's actually the title--it's referred to as "metal cubed" in common language.

[the remark this answers is not in the document]
Comment (johnwilkins) [reply release_notes/ocp-4-16-release-notes.adoc]: Possessive note fixed.

[the remark this answers is not in the document]
Comment (jeana-redhat) [reply release_notes/ocp-4-16-release-notes.adoc]: > That's actually the title--it's referred to as "metal cubed" in common language. See https://book.metal3.io/introduction

haha amazing - thanks! :sweat_smile: :facepalm:

**Comment by jab-rh** [comment]
@jeana-redhat, didn't we add in `_{context}` to all headers going forward?

**Comment by jeana-redhat** [comment]
> @jeana-redhat, didn't we add in `_{context}` to all headers going forward?

yep, my bad. RN have historically been an outlier but should now be in compliance

**Comment by JoeAldinger** [comment]
qe approved via slack.

**Comment by apinnick** [comment]
I think this needs to be reverted because the feature does not work. See HCIDOCS-371.

**Pull request merged**